truss/tests/contexts/image_builder/test_serving_image_builder.py

```diff
@@ -340,3 +341,41 @@ def test_model_cache_dockerfile():
         with open(tmp_path / "Dockerfile", "r") as f:
             gen_docker_file = f.read()
             assert secret_mount in gen_docker_file
+
+
+def test_ignore_files_during_build_setup(custom_model_truss_dir):
+    th = TrussHandle(custom_model_truss_dir)
+    truss_path = Path(th.spec.truss_dir)
+
+    ignore_files = ["file1.txt", "file2.txt", "file3.txt"]
+    ignore_folder = "ignore_folder/"
+
+    builder_context = ServingImageBuilderContext
+    image_builder = builder_context.run(th.spec.truss_dir)
+
+    # adding files to be ignored to the truss directory
```

Comment:
nit: you can move this truss setup to a fixture called `custom_model_truss_dir_with_ignored_files` to keep the test shorter